src/main/java/mate/academy/dao/MovieSessionDao.java

```diff
@@ -6,9 +6,13 @@
 import mate.academy.model.MovieSession;
 
 public interface MovieSessionDao {
-    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
+    MovieSession add(MovieSession movieSession);
 
     Optional<MovieSession> get(Long id);
 
-    MovieSession add(MovieSession session);
+    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
+
+    boolean update(MovieSession movieSession);
```

Comment:
The return type of the `update` method should be `boolean` instead of `Boolean` to avoid unnecessary boxing.